Makefile

```diff
@@ -55,7 +55,14 @@ pylint:
 
 pep8:
 	@echo "*** Running pep8 compliance check ***"
-	python3-pep8 --ignore=E501,E402,E731 blivetgui/ tests/ blivet-gui blivet-gui-daemon
+	@if test `which python3-pep8` ; then \
```

Comment:
Might be more elegant to use ``type`` instead of a combination of ``test`` and ``which`` here.